group_vars/nfss.yml.sample

```diff
@@ -44,6 +44,11 @@ dummy:
 ######################
 #ceph_nfs_log_file: "/var/log/ganesha/ganesha.log"
 #ceph_nfs_dynamic_exports: false
+# if set to true then rados is used to store dynamic exports
```

Comment:
s/dynamic exports/ganesha exports?

Why are exports stored in RADOS being called dynamic?

Reply:
fixed